vllm/engine/arg_utils.py

```diff
@@ -850,13 +852,29 @@ def add_cli_args(parser: FlexibleArgumentParser) -> FlexibleArgumentParser:
             'priority (lower value means earlier handling) and time of '
             'arrival deciding any ties).')
 
+        parser.add_argument('--pooling-type',
+                            type=str,
+                            default='CLS',
+                            choices=['LAST', 'ALL', 'CLS', 'MEAN'],
+                            help='Configures the pooling operation which '
+                            'only applies to sentence-transformers models. ')
+
+        parser.add_argument('--normalize',
+                            type=bool,
+                            default=None,
+                            help='Wheter to normalize the pooled data.')
+
         return parser
 
     @classmethod
     def from_cli_args(cls, args: argparse.Namespace):
         # Get the list of attributes of this dataclass.
         attrs = [attr.name for attr in dataclasses.fields(cls)]
         # Set the attributes from the parsed arguments.
+        pooling_config = PoolingConfig(pooling_type=args.pooling_type,
```

Comment:
Isn't `pooling_type` an `str` here? Perhaps a conversion to enum type is missing.